config/config.go

```diff
@@ -541,6 +545,7 @@ func NewConfig() *Config {
 	flag.StringVar(&cfg.OidcSecretsFile, "oidc-secrets-file", "", oidcSecretsFileUsage)
 	flag.Var(cfg.CredentialPaths, "credentials-paths", credentialPathsUsage)
 	flag.DurationVar(&cfg.CredentialsUpdateInterval, "credentials-update-interval", defaultCredentialsUpdateInterval, credentialsUpdateIntervalUsage)
+	flag.StringVar(&cfg.MaskOAuthRealms, "mask-oauth-realms", defaultMaskOAuthRealms, maskOAuthRealmsUsage)
```

Comment:
As an alternative to the flag, tokeninfo filter could store `realm`ed `auth-user` like:
```go
ctx.StateBag()["auth-user" + realm] = uid
```
which then could be used like:
```
stateBagToTag("auth-user/employee", "client_id")
``` 
or chained as a sort of allowlist:
```
stateBagToTag("auth-user/employee", "client_id") -> stateBagToTag("auth-user/whatever", "client_id")
```

Reply:
seems rather hard to use IMO